vendors/renesas/amazon_freertos_common/entropy_hardware_poll.c

```diff
@@ -19,15 +19,15 @@ int mbedtls_hardware_poll( void *data,
                            unsigned char *output, size_t len, size_t *olen )
 {
     R_INTERNAL_NOT_USED(data);
-    R_INTERNAL_NOT_USED(len);
 
     uint32_t random_number = 0;
+    size_t num_bytes = ( len < 4 ) ? len : 4;
```

Comment:
Would it make more sense to use sizeof( uint32_t ) above?  Or to use 4 below?  Seems like maybe they ought to be the same.

Reply:
yes good thinking.